README.md

```diff
@@ -36,6 +36,17 @@ http_archive(
 load("@io_bazel_rules_scala//scala:scala.bzl", "scala_repositories")
 scala_repositories()
 ```
+In addition, you **must** register `scala_toolchain` - To register default empty toolcahin simply add those lines to `WORKSPACE` file:
+```python
+
+load("@io_bazel_rules_scala/scala:toolchains.bzl", "scala_register_toolchains")
+scala_register_toolchains()
+```
+[read more here](#scala_toolchain)
+# register default scala toolchain
```

Comment:
why do we need this section? it looks like the two lines above and it doesn't seem formatted in the "view" mode